api/v2/Routes/students/masterymapping/index.js

```diff
@@ -0,0 +1,75 @@
+import { Router } from 'express';
+import {
+    getMaxScores,
+    getStudentScores,
+} from '../../../../lib/redisHelper.mjs';
+import ProgressReportData from '../../../../assets/progressReport/CS10.json' assert { type: 'json' };
```

Comment:
Will this only work for CS10 then?

Reply:
Yes. Does GV store the name of the course it is being used for? If so, I can use that.